server/src/routines/game.ts

```diff
@@ -364,6 +367,7 @@ function* turnActionSaga(
 				'PICK_REQUEST',
 				'MODAL_REQUEST',
 				'END_TURN',
+				'DELAY',
```

Comment:
This should be removed, it cancels `DELAY` actions unless it is in `availableActions` and does the next action immediately

Reply:
Fixed